policies/environments/expected.rego

```diff
@@ -38,6 +38,7 @@ expected =
     "sprinkler",
     "testing",
     "tariff",
-    "xhibit-portal"
+    "xhibit-portal",
+    "tipstaff"
```

Comment:
I have a feeling this needs to be added in an alphabetical order...

Reply:
I'll amend